api/entityapi/actions.py

```diff
@@ -38,7 +38,8 @@ def parse_entity_to_entry(
             try:
                 address = Web3.toChecksumAddress(cast(str, vals))
             except Exception:
-                logger.info(f"Unknown type of web3 address {address}")
+                logger.info(f"Unknown type of web3 address {vals}")
```

Comment:
logger.error?

Reply:
Idea was to bypass this value and just notify us, that it is not ethereum like address, without errors